[2024-10-15T19:35:24.109+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-15T19:35:24.235+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-09-15T00:00:00+00:00 [queued]>
[2024-10-15T19:35:24.305+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-09-15T00:00:00+00:00 [queued]>
[2024-10-15T19:35:24.425+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-15T19:35:24.540+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): hello_world_task2> on 2024-09-15 00:00:00+00:00
[2024-10-15T19:35:24.568+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3251) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T19:35:24.578+0000] {standard_task_runner.py:63} INFO - Started process 3293 to run task
[2024-10-15T19:35:24.579+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task2', 'scheduled__2024-09-15T00:00:00+00:00', '--job-id', '708', '--raw', '--subdir', 'DAGS_FOLDER/example_hello_world.py', '--cfg-path', '/tmp/tmplkpycira']
[2024-10-15T19:35:24.592+0000] {standard_task_runner.py:91} INFO - Job 708: Subtask hello_world_task2
[2024-10-15T19:35:24.864+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-09-15T00:00:00+00:00 [running]> on host f9b6cc3118f7
[2024-10-15T19:35:25.194+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='hello_world_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-09-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-15T00:00:00+00:00'
[2024-10-15T19:35:25.200+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-15T19:35:25.202+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-15T19:35:25.205+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World! Now I am triggered after the first task!'"]
[2024-10-15T19:35:25.229+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T19:35:25.232+0000] {subprocess.py:93} INFO - Hello World! Now I am triggered after the first task!
[2024-10-15T19:35:25.233+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T19:35:25.236+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-15T19:35:25.303+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=hello_world_task2, run_id=scheduled__2024-09-15T00:00:00+00:00, execution_date=20240915T000000, start_date=20241015T193524, end_date=20241015T193525
[2024-10-15T19:35:25.378+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-15T19:35:25.451+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T19:35:25.464+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
